portfolio/src/main/webapp/index.html

```diff
@@ -42,74 +40,89 @@ <h5 class="text-white h4">Menu</h5><br>
         currently a STEP intern at Google. This is my portfolio.
       </div>
     </div>
-
     <div class="row peach">
       <div class="col center-text">
         <img src="images.html/googlepicture.jpg" alt="Picture of Chris" style="width:7cm;height:7cm;">
       </div>
     </div>
-  </div>
-
-  <div class="row peach">
-    <div class="col center-text">
-      Below are some cool links
-    </div>
-  </div>
-
-  <div class="row peach">
-    <div class="col center-text">
-      <a class="image-link" href="https://www.linkedin.com/in/christopher-mohri-3429841a0/">
-      <img alt="LinkedIn Profile" src="images.html/linkedinlogo3.png" style="width:4cm;height:4cm;">
-      </a>  
-      <a class="image-link" href="https://github.com/chrismohrii">
-      <img alt="GitHub Profile" src="images.html/githubtransparent.png" style="width:3cm;height:3cm;">
-      </a>
-      <a class="image-link" href="https://www.tennisrecruiting.net/player.asp?id=848956">
-      <img alt="Tennisrecruiting Profile" src="images.html/tennisball2.png" style="width:3.7cm;height:3cm;">
-      </a> 
-    </div>
-  </div> 
-
-  <div class="row peach">
-    <div class="col center-text">
-      Click on these buttons for random messages 
+    <div class="row peach">
+      <div class="col center-text">
+        Below are some cool links
+      </div>
     </div>
-  </div>
-
-  <div class="row peach">
-    <div class="col-3">
+    <div class="row peach">
+      <div class="col center-text">
+        <a class="image-link" href="https://www.linkedin.com/in/christopher-mohri-3429841a0/">
+        <img alt="LinkedIn Profile" src="images.html/linkedinlogo3.png" style="width:4cm;height:4cm;">
+        </a>  
+        <a class="image-link" href="https://github.com/chrismohrii">
+        <img alt="GitHub Profile" src="images.html/githubtransparent.png" style="width:3cm;height:3cm;">
+        </a>
+        <a class="image-link" href="https://www.tennisrecruiting.net/player.asp?id=848956">
+        <img alt="Tennisrecruiting Profile" src="images.html/tennisball2.png" style="width:3.7cm;height:3cm;">
+        </a> 
+      </div>
+    </div> 
+    <div class="row peach">
+      <div class="col center-text">
+        Click on these buttons for random messages 
+      </div>
     </div>
-    <div class="col-3 right-text">
-      <button class="btn btn-outline-primary" onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="row peach">
+      <div class="col-3">
+      </div>
+      <div class="col-3 right-text">
+        <button class="btn btn-outline-primary" onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+      </div>
+      <div class="col-3 left-text">
+        <button class="btn btn-outline-primary" onclick="addRandomFunFact()">Fun Fact <br></button>
+        <div id="fact-container"></div> 
+      </div>
+      <br><br><br>
```

Comment:
Throughout this file, consider replacing usage of `<br>` with  `margin` and `padding` CSS properties in your stylesheet.

Reply:
Fixed, I kept them where I genuinely need a need line